frontend/src/lib/Navbar.svelte

```diff
@@ -56,4 +63,22 @@
         stroke: var(--primary-grey);
         height: 2rem;
     }
-</style>
\ No newline at end of file
+
+    select {
+        background-color: var(--yellow);
+        color: var(--primary-grey);
+        font-family: sans-serif, 'Ubuntu', sans-serif;
```

Comment:
? this will always be sans-serif, maybe move Ubuntu in front

Reply:
I tried removing the  2 "sans-serif" and weirdly enough, it has an effect on the font style (other than adding serifs!). Anywho I prefer the font style when only keeping the first "sans-serif", I will submit this in my next PR. Please don't hesitate to remove it if you prefer the style without it.